packages/terra-hookshot/src/Hookshot.jsx

```diff
@@ -73,12 +73,20 @@ const propTypes = {
    */
   onPosition: PropTypes.func,
   /**
-   * Required element that the content will hookshot to.
+   * Client coordinates to serve as the anchor point for the hookshot'd content.
    */
-  targetRef: PropTypes.func.isRequired,
+  targetCoordinates: PropTypes.shape({
+    x: PropTypes.number,
```

Comment:
Although it wouldn't make sense to provide just an `x` or just an `y` coordinate, should we make each key required?

Reply:
Good call